Phenotypes.pm

```diff
@@ -258,8 +278,37 @@ sub run {
 
   return {} unless $data && scalar @$data;
 
+  return { PHENOTYPES =>  $data } if $self->{config}->{output_format} eq "json";
+
+  if ($DEFAULTS{phenotype_feature}){
+    my %result_uniq;
+    my @result_str = ();
+
+    foreach my $tmp_data(@{$data}) {
+      # get required data
+      my %tmp_return =
+        map {$_ => $tmp_data->{$_}}
+        grep {defined($cols{$_})}  # only include selected cols
+        keys %$tmp_data;
+
+      # replace link characters with _
+      $tmp_return{phenotype} =~ tr/ ;,)(/\_\_\_\_\_/;
```

Comment:
Number of characters don't match on each side of the regex. Is that expected ?

Reply:
The number should match, 5/5, there is a space in the first group.